test/sanity/pylint/plugins/blacklist.py

```diff
@@ -117,6 +117,9 @@ class AnsibleBlacklistChecker(BaseChecker):
         # see https://docs.python.org/2/library/tempfile.html#tempfile.mktemp
         'tempfile.mktemp': BlacklistEntry('tempfile.mkstemp'),
 
+        # see https://github.com/ansible/ansible/pull/56785
+        'pipes.quote': BlacklistEntry('pipes.quote'),
+
```

Comment:
I just tested this and discovered that pylint isn't going to be able to check this for us. It turns out that in newer versions of Python `pipes.quote` references `shlex.quote`, so that's what pylint will see:

https://github.com/python/cpython/blob/561612d8456cfab5672c9b445521113b847bd6b3/Lib/pipes.py#L65

If we're going to block this we'll need to do so with a code-smell rule using a regex instead.

Reply:
Additionally, a regex will be difficult since it will also match our fallback code in ansible-test.

There were few enough occurrences of this, and it's easy enough to correct later that we're probably better off not bothering with a test right now, given it's more difficult than I anticipated. Once this is actually deprecated we'll be able to block that more easily (turning deprecation warnings into errors during tests).

Reply:
OK, closing.